src/StdChains.sol

```diff
@@ -199,12 +199,20 @@ abstract contract StdChains {
         setChainWithDefaultRpcUrl(
             "mainnet", ChainData("Mainnet", 1, "https://eth-mainnet.alchemyapi.io/v2/pwc5rmJhrdoaSEfimoKEmsvOjKSmPDrP")
         );
+        // @notice Goerli is to be sunset by April 3, 2024
+        // @depreciated goerli
         setChainWithDefaultRpcUrl(
             "goerli", ChainData("Goerli", 5, "https://goerli.infura.io/v3/b9794ad1ddf84dfb8c34d6bb5dca2001")
         );
```

Comment:
Sunset is soon enough (I believe April 13 per https://blog.ethereum.org/2023/11/30/goerli-lts-update) that I'm ok removing it now, since other providers/services have already dropped support for it